frontend/public/components/graphs/query-browser.tsx

```diff
@@ -37,6 +37,12 @@ export const graphColors = [
   '#1f77b4', '#ff7f0e', '#2ca02c', '#d62728', '#9467bd', '#8c564b', '#e377c2', '#7f7f7f', '#bcbd22', '#17becf',
 ];
 
+const NoQueryMessage = () => <div className="text-center text-muted">Enter a query in the box below to explore the metrics gathered for this cluster</div>;
+
+const Error = ({error}) => <div className="alert alert-danger">
```

Comment:
@rebeccaalpert FYI, new inline alert. (I know you're busy converting these to pf4.)